backend/app/services/adapters/team_kinds.py

```
@@ -767,5 +864,201 @@ def _update_team_references_in_tasks(self, db: Session, old_name: str, old_names
+    def get_team_input_parameters(
+        self, db: Session, *, team_id: int, user_id: int
+    ) -> Dict[str, Any]:
+        """
+        Get input parameters required by the team's external API bots
+        Returns parameter schema if team has external API bots, otherwise empty
+        """
+        # Get team details
+        team = db.query(Kind).filter(
+            Kind.id == team_id,
+            Kind.kind == "Team",
+            Kind.is_active == True
```

Ruff (0.14.6)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)